source/views/dictionary/list.tsx

```diff
@@ -70,8 +71,10 @@ function DictionaryView(): JSX.Element {
 		})
 	}, [navigation])
 
-	let {data, error, reload, isPending}: AsyncState<Array<WordType>> =
-		useAsync(fetchDictionaryTerms)
+	let {data, error, reload, isPending}: AsyncState<Array<WordType>> = useAsync(
```

Comment:
Isn't there an isInitial field given back to us here?

Reply:
Yes, but it does not seem to work the way we think it should. 

Using the one provided by useAsync, it looks like it never hits the loading state and initially renders the empty component message. I found this pattern of a separate `isInitialFetch` check already in-place in Student Orgs and it is working over there, too.

I agree with you. I would expect useAsync's to give us the same result, though...

Reply:
Based upon what we discovered in https://github.com/StoDevX/AAO-React-Native/pull/6021/commits/90baf4e04d0b45862f801c37b8a82a21ae4df9e8, we can use `isLoading` inside of the ListEmptyComponent.

We can refactor the dictionary list to be inline with orgs/hours once #6021 is merged. Does that sound okay?